packages/cubejs-query-orchestrator/src/config.ts

```diff
@@ -0,0 +1,24 @@
+function envIntWithDefault(envVariable: string, defaultValue: number) {
```

Comment:
It's better to use `getEnv` from `@cubejs-backend/shared` instead of doing one more place where we are getting and verifying env variables.

Reply:
Done. (I did not find it before you pointed me to it - tnx)